text/0000-buildpack-author-subteam.md

```diff
@@ -0,0 +1,125 @@
+
+As such, `libcnb` currently doesn't have well-defined processes around how it should be maintained and improved upon in order to increase adoption or serve the needs of existing users.  This leads to it feeling like an afterthought even though it is the only "official" language binding provided by the project. This lack of process hasn't stopped a [reasonable number](https://pkg.go.dev/github.com/buildpacks/libcnb?tab=importedby) of buildpack implementations from using it.
+
+Also, our user research has indicated that authoring a buildpack is difficult, and many potential buildpack authors are not go developers. In order to grow the number of buildpack authors there needs to be alternative language bindings whether provided by the project or fostered from the community:
+
+- [python](https://github.com/samj1912/python-libcnb)
+- [bash](https://github.com/jkutner/libcnb.bash)
+- [rust](https://github.com/Malax/libcnb)
+- [community go alternative](https://github.com/paketo-buildpacks/packit)
+
+Ultimately, there needs to be a group of people focused on addressing this need of making buildpack authorship more approachable.
+
+# What it is
+[what-it-is]: #what-it-is
+
+The high level charter of the proposed Buildpack Authors subteams is to help buildpack authors create buildpacks.
+
+More concretely the sub-team  would serve the following goals:
+1. Create a community focused on the needs of buildpack authors.
+1. Advocate for Buildpack Author needs from the community throughout the project
+1. Devote attention to improving our existing buildpack-author tooling (`libcnb`).
+1. Provide a home for any future buildpack author tooling (testing buildpacks and language buildpack templates).
+1. Encourage buildpack authors to get more involved in and contribute to the project by creating a team exclusively focused on building more shared tooling for buildpack authors.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+## Sub-team Guidelines
+
+The Buildpack Author sub-team will follow the same guidelines as other sub-teams:
+
+- include at least one core team member
+- responsible for narrower sets of concerns related to specific aspects of the project
+
+## Language Binding Responsibilties
+These are the responsibilities for maintaining any language binding:
+1. Implementing new APIs in a timely manner
+1. Release process, planning and updates
+1. Triaging issues and supporting users
+1. Documentation
+1. Buildpack Examples using these bindings
+1. Planning and executing improvements
+
+Today, this means taking ownership over the `libcnb` repo doing the above for it.
```

Comment:
I wonder if it might be beneficial to have a `staging` org for language bindings. This would give us the opportunity to see if they are mature enough to satisfy at the above requirements & also give them some additional visibility.

Reply:
@dwillist a good thought and maybe not specific to language bindings. I would think any code base coming in would make sense here and if it's something we create how do we experiment / test something without it requiring the same level of rigor we have for our existing software like `pack`?

Reply:
Should we also call this out in the rfc for accepting project level contributions once it's put in (I assume) the community repo? https://github.com/buildpacks/rfcs/pull/143

Reply:
@samj1912 That sounds like the perfect place IMO.